interface/patient_file/encounter/forms.php

```diff
@@ -632,10 +638,22 @@ function divtoggle(spanid, divid) {
         echo "<div class='form_header_controls'>";
 
         // If the form is locked, it is no longer editable
-        if ( $esign->isLocked() ) {
-            echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-".attr($formdir)."-".attr($iter['id'])."'><span>".xlt('Locked')."</span></a>";
+        if ($esign->isLocked()) {
+             $flag = false;
+             if($GLOBALS['enable_group_therapy']){
+                 if($acl_groups) $flag = true;
+             }
+             else{
+                 $flag = true;
+             }
+
+             if($flag){
+                 echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-" . attr($formdir) . "-" . attr($iter['id']) . "'><span>" . xlt('Locked') . "</span></a>";
+             }
```

Comment:
Also don't understand this. There is no test if the form is related to groups. What is the intent here?